bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/RadioButton.java

```diff
@@ -0,0 +1,27 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2022 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ConsumerType;
+
+/**
+ * Defines the form {@code RadioGroup} Sling Model used for the
+ * {@code /apps/core/fd/components/form/radiogroup/v1/radiogroup} component.
+ *
+ * @since com.adobe.cq.forms.core.components.models.form 2.0.0
+ */
+@ConsumerType
+public interface RadioButton extends CheckBox {}
```

Comment:
pls remove it not required.

Reply:
didn't get this, shouldn't we create a separate class for RadioButton for better readability

Reply:
as discussed over teams, this is required